provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -243,19 +244,35 @@ def _create_node(self, machine_id: str) -> NodeDesc:
         hostname = store.get(f'server_node>{machine_id}>hostname')
         name = store.get(f'server_node>{machine_id}>name')
         iface = self._get_iface(machine_id)
+        servers = DList([
```

Comment:
Please add some documentation, it would be good for a reader. Something like,
```
# Create 1 io_disks entry in CDF per cvg. An io_disks entry comprises of a set of data devices and a meta data device.
```
On second thought, I think it would be good to simplify this by adding functions,
`get_data_devices(machine_id: str, cvg: int) -> List[str]` and `get_meta_data_devices(machine_id: str, cvg: int) -> List[str]`

Reply:
Done